rollup.config.js

```diff
@@ -0,0 +1,28 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import resolve from 'rollup-plugin-node-resolve';
+import replace from 'rollup-plugin-replace';
+
+const NODE_ENV = process.env.NODE_ENV || 'development';
+const outputFile = NODE_ENV === 'production' ? './lib/prod.js' : './lib/dev.js';
```

Comment:
I'm not sure to get why we need to rely on specific environements here? I'm not that good with building tools 😓

Reply:
Sometimes, bundled code works differently if built for dev or production. As a result, I actually compile two binaries. Prod.js and Dev.js. Often, prod dependencies log nothing related to debugging.

Your code isn't actually the reason that I'm doing this, it's because of the dependencies which in this case is only `prop-types`. I'm unsure if proptypes (or any of it's dependencies) require this. It's often best practice to just put it in something that in. The rollup config I use on my FOSS projects is pretty standard.